.reuse/dep5

```diff
@@ -353,10 +357,6 @@ Files: libs/type_safe/*
 Copyright: 2016-2019, Jonathan Müller
 License: MIT
 
-Files: libs/type_safe/external/debug_assert/*
```

Comment:
Why removing this?

Reply:
Copyright and license appears to be already described by `libs/type_safe/*`

Reply:
But it's not the same license. type_safe is MIT, while debug_assert is Zlib.

Reply:
You are right! Confusingly, the `libs/type_safe/external/debug_assert/CMakeLists.txt` file mentions a LICENSE file but I verified on the upstream repo - it's still under Zlib. Updated.